test/project_finder/project_finder_integration_test.dart

```diff
@@ -15,13 +15,14 @@ void main() {
     );
 
     // then
-    expect(projects, hasLength(6));
+    expect(projects, hasLength(7));
```

Comment:
These expects are hard to read and have a lot of occurrences. What do you think about helper method like `expectProjectsWithNames(List<Projects>, List<String>)` ?